packages/camera/camera_android_camerax/test/camera_test.dart

```diff
@@ -18,9 +18,17 @@ void main() {
         instanceManager: instanceManager,
       );
 
-      flutterApi.create(0);
+      // InstanceManagerHostApi.clear() is async. It is called unawaited deep inside of `flutterApi.create(0)`.
```

Comment:
This doesn't seem correct. `CameraFlutterApiImpl` does not include  a reference to `JavaObject. globalInstanceManager` when passed a nonnull `InstanceManager`: https://github.com/flutter/packages/blob/main/packages/camera/camera_android_camerax/lib/src/camera.dart#L25

Reply:
Actually it looks like this line needs to be removed since this is the line causing the call to clear: https://github.com/flutter/packages/blob/main/packages/camera/camera_android_camerax/lib/src/camera.dart#L20.